tests/configurations/unit/jest.config.js

```diff
@@ -32,8 +32,8 @@ module.exports = {
   testEnvironment: 'node',
   coverageThreshold: {
     global: {
-      branches: 74,
-      functions: 90,
+      branches: 73,
```

Comment:
try revert and run tests. 
please create an issue - 'Exporter-trigger: add missing tests.'